assimilation_code/modules/assimilation/filter_mod.f90

```diff
@@ -308,8 +305,7 @@ module filter_mod
    output_mean,                  &
    output_sd,                    &
    write_all_stages_at_end,      &
-   write_obs_every_cycle,        & 
-   allow_missing_clm
+   write_obs_every_cycle
```

Comment:
personally i love this change.  i'm just checking that this has been discussed in a software or group meeting?  in the past we've worked really hard to avoid non-backwards compatible namelist changes.   honestly, it probably complicates the code more than is worthwhile.  

but we have left in obsolete items for a few releases before removing them.  we add code to check to see if someone is setting the namelist value and print a 'this item is obsolete, we are ignoring it' message.   

if someone has this in their filter namelist, the current namelist read code will print out a fatal error saying this item is unrecognized.  if the release notes say this is being removed and is a non-backwards compatible change, i'm good with it.  i'm just checking what the current rules are here.

Reply:
reverted filter_mod.f90  match main.